tests/test_scripts.py

```diff
@@ -412,27 +413,36 @@ def test_success(self):
         library = self._library()
         s = SetCoverageAreaScript(_db=self._db)
 
-        # Setting a service area with no focus area assigns that
+        # Setting an eligibility area with no focus area assigns that
         # service area to the library.
         args = ["--library=%s" % library.name,
                 '--service-area={"US": "everywhere"}']
         s.run(args)
         [area] = library.service_areas
         assert area.place == us
+        assert area.type == ServiceArea.FOCUS
```

Comment:
This is a little tricky because the command-line argument is `--service-area` but internally we call this an "eligibility area".